src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+$maincolor: #060b35;
+$secondaryColor: #616070;
+$buttonColor: #00acdc;
+
+html {
+  font-family: "Roboto", "Helvetica", sans-serif;
+  color: $maincolor;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
```

Comment:
You don't need to specify margin-top: 0;
If some unnecessary styles you get from elements you can reset them with reset.css or manually.
Also, if you have different margin or padding properties for one class, use shorthand properties instead of specifying both top and bottom.